src/screens/OtherProfile/index.js

```diff
@@ -652,14 +661,14 @@ const styles = StyleSheet.create({
   wrapImageProfile: {
     marginTop: 24,
     flexDirection: 'column',
-    backgroundColor: 'red'
+    backgroundColor: COLORS.redalert
   },
   nameProfile: {
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 17,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `backgroundColor` property from `'red'` to `COLORS.redalert` is a good move towards using a consistent color scheme throughout the application. However, it's important to ensure that the `redalert` color is defined correctly in the `COLORS` object and that it doesn't affect the visibility or aesthetics of the `wrapImageProfile` element.



<!-- This is an auto-generated comment by OSS CodeRabbit -->